cmd/github_receiver/main.go

```diff
@@ -47,6 +47,7 @@ var (
 	receiverAddr    = flag.String("webhook.listen-address", ":9393", "Listen on address for new alertmanager webhook messages.")
 	alertLabel      = flag.String("alertlabel", "alert:boom:", "The default label applied to all alerts. Also used to search the repo to discover exisitng alerts.")
 	extraLabels     = flagx.StringArray{}
+	titleTmplFile   = flagx.FileBytes{}
```

Comment:
Please set the default value here using something like:

```
titleTmplFile = flagx.FileBytes(alerts.DefaultTitleTmpl)
```

This will make it possible to eliminate the branch in `alerts.NewReceiver`:

```
	if titleTmplStr == "" {
		titleTmplStr = DefaultTitleTmpl
	}
```

Reply:
Yes, I was ready for another review; guess I'm not familiar with the reviewable.io interface. Thanks for continuing to look at it. Here's the latest, with your suggestion incorporated.